itou/templates/admin/users/transfer_user.html

```diff
@@ -0,0 +1,96 @@
+{% extends "admin/base_site.html" %}
+{% load i18n admin_urls static %}
+
+{# Adapted from Django's admin/change_form.html #}
+{% block extrahead %}
+    {{ block.super }}
+    <script src="{% url 'admin:jsi18n' %}"></script>
+    {{ media }}
+    <script nonce="{{ CSP_NONCE }}">
+        document.addEventListener('DOMContentLoaded', function() {
+            var buttons = document.getElementsByClassName("with-confirm");
```

Comment:
Allez, un peu de modernisation.
```suggestion
            const buttons = document.getElementsByClassName("with-confirm");
```